pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -124,23 +124,23 @@ type ClusterBuilder struct {
 // NewClusterBuilder builds an instance of ClusterBuilder.
 func NewClusterBuilder(proxy *model.Proxy, req *model.PushRequest, cache model.XdsCache) *ClusterBuilder {
 	cb := &ClusterBuilder{
-		serviceInstances:  proxy.ServiceInstances,
-		proxyID:           proxy.ID,
-		proxyType:         proxy.Type,
-		proxyVersion:      proxy.Metadata.IstioVersion,
-		sidecarScope:      proxy.SidecarScope,
-		passThroughBindIP: getPassthroughBindIP(proxy),
-		supportsIPv4:      proxy.SupportsIPv4(),
-		supportsIPv6:      proxy.SupportsIPv6(),
-		hbone:             proxy.EnableHBONE(),
-		locality:          proxy.Locality,
-		proxyLabels:       proxy.Labels,
-		proxyView:         proxy.GetView(),
-		proxyIPAddresses:  proxy.IPAddresses,
-		configNamespace:   proxy.ConfigNamespace,
-		req:               req,
-		cache:             cache,
-	}
+		serviceInstances: proxy.ServiceInstances,
+		proxyID:          proxy.ID,
+		proxyType:        proxy.Type,
+		proxyVersion:     proxy.Metadata.IstioVersion,
+		sidecarScope:     proxy.SidecarScope,
+		supportsIPv4:     proxy.SupportsIPv4(),
+		supportsIPv6:     proxy.SupportsIPv6(),
+		hbone:            proxy.EnableHBONE(),
+		locality:         proxy.Locality,
+		proxyLabels:      proxy.Labels,
+		proxyView:        proxy.GetView(),
+		proxyIPAddresses: proxy.IPAddresses,
+		configNamespace:  proxy.ConfigNamespace,
+		req:              req,
+		cache:            cache,
+	}
+	cb.passThroughBindIPs = getPassthroughBindIPs(proxy.GetIPMode())
```

Comment:
very nit: there is no need to split this out, just causes confusion.

Reply:
okay, I will merge it in later PR